cli/admin/README.md

```diff
@@ -65,18 +65,19 @@ Pravega Admin CLI.
    
 Initial configuration:
     cli.store.metadata.backend=segmentstore
-    cli.controller.connect.credentials.username=admin
+    cli.credentials.username=admin
```

Comment:
Why are we changing these?

Reply:
I made the change to make the use of credentials for pravega cli as a whole rather than only the controller in the pravega cli. Is that ok?

Reply:
Ok. I eventually realized as I was reading down the PR last time.